win32ss/user/user32/include/user32.h

```diff
@@ -72,4 +72,8 @@ typedef struct _PATRECT
     HBRUSH hBrush;
 } PATRECT, * PPATRECT;
 
+HRESULT
+GetImmFileName(_Out_ LPWSTR lpBuffer,
+               _In_ size_t cchBuffer);
```

Comment:
Please move that definition to this file instead:
https://git.reactos.org/?p=reactos.git;a=blob;f=win32ss/user/user32/include/user32p.h;hb=a8d2cd4b9d124bf9a9d97105cb11923827d7142b#l131

Reply:
Done.
https://github.com/reactos/reactos/commit/4689c8ead89e97ad6b2bcf0a7fc88210d9ca285c